protobuf/api/instance.proto

```diff
@@ -1,12 +1,23 @@
 syntax = "proto3";
 
+import "protobuf/definition/instance.proto";
+
 package api;
 
 service Instance {
+  rpc List (ListInstancesRequest) returns (ListInstancesResponse) {}
   rpc Create (CreateInstanceRequest) returns (CreateInstanceResponse) {}
   rpc Delete (DeleteInstanceRequest) returns (DeleteInstanceResponse) {}
 }
 
+message ListInstancesRequest {
+  string serviceHash = 1;   // service hash of instance to filter by service (optional).
```

Comment:
This filter is actually not optional.
There is no test on its default value in `sdk/instance/instance.go` or `database/instance_db.go`

Reply:
do you want to make it optional or rm the optional word from proto?

Reply:
make it optional ;)

Reply:
done